src/libraries/System.Private.Xml/src/System/Xml/Xsl/IlGen/XmlILOptimizerVisitor.cs

```diff
@@ -3271,7 +3268,6 @@ protected override QilNode VisitFilter(QilLoop local0)
         protected override QilNode VisitSort(QilLoop local0)
         {
             QilNode local1 = local0[0];
-            QilNode local2 = local0[1];
```

Comment:
Is there any chance that these indexing operations might have thrown, and that was somehow important?

Reply:
I don't think so, `QilLoop` never throws for 0 and 1 indexes. https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/System.Private.Xml/src/System/Xml/Xsl/QIL/QilBinary.cs#L51

It looks more like copy&paste error.